projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -78,12 +79,23 @@ export class CartItemListComponent {
         return entry;
       });
     } else {
-      this._items = items;
+      if (this.featureConfigService?.isLevel('3.1')) {
```

Comment:
Can we also bump the feature level in app.module.ts to `3.1`, so the new behavior becomes the default (btw. e2e will cover the new behavior)?